roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -13,7 +13,10 @@
   vars:
     # We will restart Docker ourselves after everything is ready:
     skip_docker_restart: True
-  when: l_docker_upgrade is defined and l_docker_upgrade | bool and not openshift.common.is_atomic | bool
+  when:
+  - l_docker_upgrade is defined
+  - l_docker_upgrade | bool
+  - not openshift.common.is_atomic | bool
```

Comment:
Not related to this PR directly, but I'm wondering if we should be skipping the docker restart based on `is_containerized` rather than `is_atomic`...